plugins/module_utils/network/ios/rm_templates/static_routes.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+"""
+The Static_routes parser templates file. This contains
+a list of parser definitions and associated functions that
+facilitates both facts gathering and native command generation for
+the given network resource.
+"""
+
+import re
+
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.rm_base.network_template import (
+    NetworkTemplate,
+)
+
+
+class Static_routesTemplate(NetworkTemplate):
+    def __init__(self, lines=None, module=None):
+        super(Static_routesTemplate, self).__init__(lines=lines, tmplt=self, module=module)
+
+    # fmt: off
+    PARSERS = [
+        {
+            "name": "ipv4",
+            "getval": re.compile(
+                r"""
+                ^ip\sroute
+                (\stopology\s(?P<topology>\S+))?
+                (\svrf\s(?P<vrf>\S+))?
+                (\s(?P<dest>\S+))
+                (\s(?P<netmask>\S+))
+                (\s(?P<interface>(ACR|ATM-ACR|Analysis-Module|AppNav-Compress|AppNav-UnCompress|Async|Auto-Template|BD-VIF|BDI|BVI|Bluetooth|CDMA-Ix|CEM-ACR|CEM-PG|CTunnel|Container|Dialer|EsconPhy|Ethernet-Internal|Fcpa|Filter|Filtergroup|GigabitEthernet|IMA-ACR|LongReachEthernet|Loopback|Lspvif|MFR|Multilink|NVI|Null|PROTECTION_GROUP|Port-channel|Portgroup|Pos-channel|SBC|SDH_ACR|SERIAL-ACR|SONET_ACR|SSLVPN-VIF|SYSCLOCK|Serial-PG|Service-Engine|TLS-VIF|Tunnel|VPN|Vif|Vir-cem-ACR|Virtual-PPP|Virtual-TokenRing)\d+))?
```

Comment:
Do we need to check for the interface type here?

Reply:
If we don't check the interface type it gets packed as dest or netmask,
`ip route 198.51.xx.x 255.255.255.0 GigabitEthernet2 198.51.yy.yy 22 tag 22 permanent name nm1 multicast `
^ this being a valid config line